cmake/modules/ConfigurePyTorch.cmake

```diff
@@ -0,0 +1,21 @@
+function(ProbeForPyTorchInstall)
+  if(Torch_ROOT)
+    message(STATUS "Using cached Torch root = ${Torch_ROOT}")
+  else()
+    message(STATUS "Checking for PyTorch using ${PYTHON_EXECUTABLE} ...")
+    execute_process(
+      COMMAND ${PYTHON_EXECUTABLE}
+      -c "import os;import torch;print(os.path.dirname(torch.__file__), end='')"
+      WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+      RESULT_VARIABLE PYTORCH_STATUS
+      OUTPUT_VARIABLE PYTORCH_PACKAGE_DIR)
+    if(NOT PYTORCH_STATUS EQUAL "0")
+      message(FATAL_ERROR "Unable to 'import torch' with ${PYTHON_EXECUTABLE}")
```

Comment:
When I tried running `./build_tools/cmake_configure.sh` without PyTorch installed, I did hit this, and it exited the script. If that is the intent, should this also be checking if PyTorch is required? I was using the default setting where PyTorch is optional, and I expected to be able to proceed without it.

Reply:
Yeah, good catch. The CI also complained to me about that and what is now landed at head does the right thing when PyTorch isn't installed. I just had a git problem and didn't update the PR properly before pushing.